code/SecurityAdmin.php

```diff
@@ -3,314 +3,145 @@
-                )
-            )->setTemplate('SilverStripe\\Forms\\CMSTabSet'),
-            // necessary for tree node selection in LeftAndMain.EditForm.js
-            new HiddenField('ID', false, 0)
-        );
-
-        // Add roles editing interface
-        $rolesTab = null;
-        if (Permission::check('APPLY_ROLES')) {
-            $rolesField = GridField::create(
-                'Roles',
-                false,
-                PermissionRole::get(),
-                GridFieldConfig_RecordEditor::create()
-            );
-
-            $rolesTab = $fields->findOrMakeTab('Root.Roles', _t(__CLASS__ . '.TABROLES', 'Roles'));
-            $rolesTab->push($rolesField);
         }
-
-        // Build replacement form
-        $form = Form::create(
-            $this,
-            'EditForm',
-            $fields,
-            new FieldList()
-        )->setHTMLID('Form_EditForm');
-        $form->addExtraClass('cms-edit-form fill-height');
-        $form->setTemplate($this->getTemplatesWithSuffix('_EditForm'));
-        $form->addExtraClass('ss-tabset cms-tabset ' . $this->BaseCSSClasses());
-        $form->setAttribute('data-pjax-fragment', 'CurrentForm');
-
-        $this->extend('updateEditForm', $form);
-
-        return $form;
-    }
-
-    public function memberimport()
-    {
-        Requirements::clear();
-        Requirements::javascript('silverstripe/admin: client/dist/js/vendor.js');
-        Requirements::javascript('silverstripe/admin: client/dist/js/MemberImportForm.js');
-        Requirements::css('silverstripe/admin: client/dist/styles/bundle.css');
-
-        return $this->renderWith('BlankPage', [
-            'Form' => $this->MemberImportForm()->forTemplate(),
-            'Content' => ' '
-        ]);
+        return $models;
     }
 
     /**
-     * @see SecurityAdmin_MemberImportForm
-     *
-     * @return Form
+     * @return Form|false
      */
-    public function MemberImportForm()
+    public function ImportForm()
     {
-        if (!Permission::check('ADMIN')) {
-            return null;
+        $form = parent::ImportForm();
+        if (!$form) {
+            return $form;
         }
-
-        /** @var Group $group */
-        $group = $this->currentPage();
-        $form = new MemberImportForm($this, __FUNCTION__);
-        $form->setGroup($group);
-
-        return $form;
-    }
-
-    public function groupimport()
-    {
-        Requirements::clear();
-        Requirements::javascript('silverstripe/admin: client/dist/js/vendor.js');
-        Requirements::javascript('silverstripe/admin: client/dist/js/MemberImportForm.js');
-        Requirements::css('silverstripe/admin: client/dist/styles/bundle.css');
-
-        return $this->renderWith('BlankPage', [
-            'Content' => ' ',
-            'Form' => $this->GroupImportForm()->forTemplate()
+        Requirements::javascript('silverstripe/admin:client/dist/js/MemberImportForm.js');
+        $form->Fields()->removeByName('EmptyBeforeImport');
```

Comment:
Why are we removing that field?
If it's just because it wasn't there in `SecurityAdmin` originally I think we should keep it here.
We should only remove it if there's a very clear reason to not provide this functionality.

Reply:
> Perhaps we could remove it for non-admins only
Non-admins aren't even given the option to bulk load from CSV

It feels extremely risky to including this checkbox because if it's accidentally checked then you'll completely destroy your content auditing, as well as the ability for anyone to login.

I don't think there's a valid use case for removing ALL members.  I can't recall any feedback on this part of the CMS saying "please add in the checkbox that lets me delete all members on csv import"

Reply:
Fair enough, I'm okay with that.